### ISSUE TITLE
Fix color contrast issues associated with primary accent color

### ISSUE THREAD

**Issue by brittanyhuntzberry** [brittanyhuntzberry/workshop-08-2024#1]
axe Issues
https://axe.deque.com/issues/d6964f5b-da2a-4816-9c4f-e7157a5409f3
https://axe.deque.com/issues/b8381f8f-54d3-4948-aa0f-c3404c546a81
Designer Notes
Use #054A91 instead to increase the contrast ratio and become WCAG 2.1 AA compliant

Dev Notes
Note

You should be able to fix this with just 1 line of CSS change -- see fictitious.css
